### PR TITLE
feat: add python Plugin tutorials content

### PR DESCRIPTION
**Pull request by jbrockopp**

### CODE REVIEW

File: content/plugins/tutorials/python.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+print(response.text.encode('utf8'))
+```
+
+## Executable
+
+Now that we have the script to accomplish our plugin's task, we need to ensure the code is executable for our plugin:
+
+```sh
+chmod +x script.py
```
Comment (wass3r) [inline content/plugins/tutorials/python.md]: This command (and section) is not really necessary, afaik.

Comment (jbrockopp) [reply content/plugins/tutorials/python.md]: You are correct 👍 

I've updated the Docker entrypoint in a new commit to ensure we reflect the nature of why we need to `chmod` the script 🎉

**Review by kneal — DISMISSED** [review]
LGTM 🐬

**Review by wass3r — APPROVED** [review]
lgtm

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Pull request merged**